pkg/users/accounts-list.js

```diff
@@ -110,6 +110,8 @@ const UserActions = ({ account }) => {
                 isPlain
                 isOpen={isKebabOpen}
                 position="right"
+                id="accounts-actions"
+                menuAppendTo={document.body}
```

Comment:
Hmm, everywhere else we write this as `menuAppendTo={() => document.body}`. But quite obviously this works too, nice! https://www.patternfly.org/v4/components/select also documents both possibilities.

However, https://www.patternfly.org/v4/components/select#appending-document-body-vs-parent discourages document.body, and suggests `"parent"` instead, for accessibility. We need to do that for all cases, but test them individually whether it actually works. Let's do that as a follow-up.